libloki/test/proof-of-work_test.js

```diff
@@ -8,88 +8,98 @@ const {
   greaterThan,
 } = pow;
 
-describe('Proof of Work Worker', () => {
-  it('should increment a Uint8Array nonce correctly', () => {
-    const arr1Before = new Uint8Array([0,0,0,0,0,0,0,0]);
-    const arr1After = incrementNonce(arr1Before);
-    assert.strictEqual(arr1After[0], 0);
-    assert.strictEqual(arr1After[1], 0);
-    assert.strictEqual(arr1After[2], 0);
-    assert.strictEqual(arr1After[3], 0);
-    assert.strictEqual(arr1After[4], 0);
-    assert.strictEqual(arr1After[5], 0);
-    assert.strictEqual(arr1After[6], 0);
-    assert.strictEqual(arr1After[7], 1);
-  });
+describe('Proof of Work', () => {
+  describe('#incrementNonce', () => {
+    it('should increment a Uint8Array nonce correctly', () => {
+      const arr1Before = new Uint8Array([0,0,0,0,0,0,0,0]);
+      const arr1After = incrementNonce(arr1Before);
+      assert.strictEqual(arr1After[0], 0);
```

Comment:
Just realised this could probably be
```
const arr1Expected = new Uint8Array([0,0,0,0,0,0,0,1]);
assert.deepStrictEqual(arr1After, arr1Expected);
```
Also maybe the other deepEquals used should be deepStrictEquals? (I assume we should always use strictEquals unless explicitly otherwise?)
Soz these were mine lol

Reply:
Oh right, i'll fix them all up 👍

Reply:
Chai assert doesn't support `deepStrictEqual` so this is not possible